src/rtext.c

```diff
@@ -1344,40 +1344,39 @@ Rectangle GetGlyphAtlasRec(Font font, int codepoint)
 // Get text length in bytes, check for \0 character
 unsigned int TextLength(const char *text)
 {
-    unsigned int length = 0; //strlen(text)
-
-    if (text != NULL)
```

Comment:
Please do not remove this check. `TextLength(NULL)` would cause a `segfault`.
Also, I believe that's the standard structure ([reference](https://github.com/raysan5/raylib/commit/b68d0850b1dee42d818e67d99f4affe1d890dba1#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R3767-R3777)).